doc/usage.rst

```diff
@@ -119,7 +119,27 @@ a list, e.g.
         '--icon=%s' % os.path.join('resource', 'path', 'icon.ico'),
         os.path.join('my_package', '__main__.py'),
     ])
+    
+If using a single file script or main function not packed in a package or namespace 
+the following namespace can be used as a skeleton in a python build file, named 
```

Comment:
I don't understand this sentence at all. You always pass a single file to PyInstaller. And a single file is always either a package or part of a package.

Reply:
```suggestion
If using a single file script or main function not packed in a package or namespace 
the following namespace can be used as a skeleton in a python build file, named 
```
I agree that this sentence has some problems. I was trying to clarify that if you only have a single python file, not packaged in a folder (my_package) this was a simpler solution.